test/jest/network.spec.ts

```diff
@@ -0,0 +1,209 @@
+    const denied_port_response = await execInPod("test-admin-app", testAdminApp, "curl", blocked_port_curl);
+    expect(denied_port_response.stdout).toBe("503");
+
+    // Wide open Egress means successful google curl
+    const successful_google_response = await execInPod("test-admin-app", testAdminApp, "curl", GOOGLE_CURL);
+    expect(successful_google_response.stdout).toBe("200");
+  });
+
+  test.concurrent("Anywhere Egress", async () => {
+    // Validate that request is successful when Egress Anywhere is used
+    const success_response = await execInPod("test-admin-app", testAdminApp, "curl", CURL_EXTERNAL);
+    expect(success_response.stdout).toBe("200");
+
+    // Validate Egress to Google is successful
+    const successful_google_response = await execInPod("test-admin-app", testAdminApp, "curl", GOOGLE_CURL);
+    expect(successful_google_response.stdout).toBe("200");
+  });
+
+  test.concurrent("RemoteNamespace Ingress and Egress", async () => {
+    // Validate that request is successful when using RemoteNamespace
+    const success_response = await execInPod("curl-test-4", curlPodName6, "curl-6", INTERNAL_CURL_COMMAND_5);
+    expect(success_response.stdout).toBe("200");
+
+    // Default Deny for Google Curl when no Egress defined
+    const denied_google_response = await execInPod("curl-test-4", curlPodName6, "curl-6", GOOGLE_CURL);
+    expect(denied_google_response.stdout).toBe("000");
+
+    // Default Deny for Blocked Port
+    const blocked_port_curl = getCurlCommand("curl-7", "curl-test-5", 9999);
+    const denied_port_response = await execInPod("curl-test-4", curlPodName6, "curl-6", blocked_port_curl);
+    expect(denied_port_response.stdout).toBe("503");
+  });
+
+  test.concurrent("Kube API Restrictions", async () => {
+    const kubeApi_curl = [
+      "curl",
+      "-s",
+      "-o",
+      "/dev/null",
+      "-w",
+      "%{http_code}",
+      "-k",
+      "-H",
+      "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)",
+      `https://kubernetes.default.svc.cluster.local/api/v1/namespaces`,
+    ];
+
+    // Validate unauthorized response when using Kube API ( because of missing authorization token )
+    const success_unauthorized_response = await execInPod("curl-test-6", curlPodName8, "curl-8", kubeApi_curl);
+    expect(success_unauthorized_response.stdout).toContain("401");
+
+    // Default Deny for Google Curl when no Egress defined
+    const denied_google_response = await execInPod("curl-test-6", curlPodName8, "curl-8", GOOGLE_CURL);
+    expect(denied_google_response.stdout).toBe("000");
+
+    // Default Deny for Blocked Port
+    const blocked_port_curl = getCurlCommand("curl-2", "curl-test-1", 9999);
+    const denied_port_response = await execInPod("curl-test-6", curlPodName8, "curl-8", blocked_port_curl);
+    expect(denied_port_response.stdout).toBe("503");
+  });
+
+  test.concurrent("RemoteCidr Restrictions", async () => {
```

Comment:
Might not be tracing things right here but I think the pod we're using for the curls here is actually a [`remoteGenerated: Anywhere`](https://github.com/defenseunicorns/uds-core/blob/network-testing/src/test/app-admin.yaml#L41)? Is that intentional/should this test name be different?